packages/react-hooks/src/recordUtils.ts

```diff
@@ -105,3 +106,15 @@ export const importModule = async (moduleName: string): Promise<any> => {
   const importedModule = await import(moduleName)
   return importedModule
 }
+
+export const checkModuleEnabled = (agent: Agent, moduleName: any) => {
+  if (!agent) {
+    throw new Error('Agent is required to check if a module is enabled')
+  }
+
+  let enabled = false
+  if (Object.values(agent.dependencyManager.registeredModules).find((module: any) => module instanceof moduleName)) {
+    enabled = true
+  }
+  return enabled
```

Comment:
Find returns the value when it find something. Although this works, it's not how it's intended to be used

```suggestion
  const foundModule = Object.values(agent.dependencyManager.registeredModules).find((module: any) => module instanceof moduleName))
  
  return foundModule !== undefined
```